modular_pentest/modules/v_outpost/code/outpost.dm

```diff
@@ -1,13 +1,6 @@
-/datum/map_template/ruin/pentest
-	prefix = "PentestModules/maps/"
-	cost = 10
-	allow_duplicates = FALSE
-
-//Made a ruin or something? put it here!
```

Comment:
Readd Old Ln 1-6 as this is probably required to function since you deleted the prefix to the suffix for the lower ruin